src/rollbar.js

```diff
@@ -14,6 +14,7 @@ var _ = require('./utility');
 function Rollbar(options, api, logger, platform) {
   this.options = _.extend(true, {}, options);
   this.logger = logger;
+  Rollbar.rateLimiter.configureGlobal(this.options);
```

Comment:
This fixes the issue where the maxItems limit was not getting set via the constructor and therefore from the global _rollbarConfig.